--- Net-SSLeay-1.78/inc/Module/Install/PRIVATE/Net/SSLeay.pm.0000~	2016-09-15 23:33:44.110358389 +0200
+++ Net-SSLeay-1.78/inc/Module/Install/PRIVATE/Net/SSLeay.pm	2016-09-15 23:34:32.893847385 +0200
@@ -67,7 +67,7 @@ sub ssleay_get_build_opts {
     for ("$prefix/include", "$prefix/inc32", '/usr/kerberos/include') {
       push @{$opts->{inc_paths}}, $_ if -f "$_/openssl/ssl.h";
     }
-    for ($prefix, "$prefix/lib64", "$prefix/lib", "$prefix/out32dll") {
+    for ("$prefix/out32dll") {
       push @{$opts->{lib_paths}}, $_ if -d $_;
     }
 
